src/lib/isInt.js

```diff
@@ -10,15 +11,15 @@ export default function isInt(str, options) {
   // Get the regex to use for testing, based on whether
   // leading zeroes are allowed or not.
   let regex = (
-    options.hasOwnProperty('allow_leading_zeroes') && !options.allow_leading_zeroes ?
+    hasOption(options, 'allow_leading_zeroes') && !options.allow_leading_zeroes ?
       int : intLeadingZeroes
   );
```

Comment:
Is `hasOption` needed here? Can't we just do this since both `undefined` and `null` will be converted to false? 
```javascript
   let regex = options.allow_leading_zeroes ? intLeadingZeroes : int;
```

Reply:
Yes we can